cmd/ddev/cmd/config.go

```diff
@@ -50,37 +50,37 @@ var ConfigCommand = &cobra.Command{
 			provider = args[0]
 		}
 
-		c, err := ddevapp.NewConfig(appRoot, provider)
+		app, err := ddevapp.NewApp(appRoot, provider)
```

Comment:
It may be more trouble than it's worth, but I think a rename of `ddevapp` to just `ddev` would make a lot of this code read better. For example

`ddevapp.NewApp()` -> `ddev.NewApp()`
`ddevapp.GetActiveApp()` -> `ddev.GetActiveApp()`

Reply:
It's easy to change, but I'm not sold on 'ddev' for the local var here, it's too confusing being linked to the project name. But something else would be OK. Maybe just change the local var usage to 'd' ?